lexers/lexers_test.go

```diff
@@ -117,3 +123,40 @@ func TestLexers(t *testing.T) {
 		}
 	}
 }
+
+func TestLexersTextAnalyser(t *testing.T) {
+	files, err := ioutil.ReadDir("testdata/analysis")
+	assert.NoError(t, err)
+
+	for _, file := range files {
+
+		ext := filepath.Ext(file.Name())[1:]
+		if ext != "actual" {
+			continue
+		}
+
+		base := strings.Split(strings.TrimSuffix(file.Name(), filepath.Ext(file.Name())), ".")[0]
+
+		t.Run(base, func(t *testing.T) {
+			lexer := lexers.Get(base)
+			assert.NotNil(t, lexer)
```

Comment:
Will pass the name :+1: Regarding require, if I understood your fork of testify/assert correctly, this will abort on assertion failure, right? https://github.com/alecthomas/assert/blame/master/README.md#L8

Reply:
Ah sorry, my bad, I forgot I was using alecthomas/assert in Chroma 🤦